code/datums/uplink_item.dm

```diff
@@ -129,6 +129,14 @@ var/list/uplink_items = list()
 	cost = 5
 	job = list("Clown")
 
+/datum/uplink_item/jobspecific/clownmagboots
+	name = "Clown Magboots"
+	desc = "A pair of modified clown shoes fitted with an advanced magnetic pull system. Look and sound exactly like regular clown shoes unless closely inspected."
```

Comment:
Perhaps 'traction' rather than 'pull'.

Reply:
All the other descriptions in magboots.dm mention the boots' "magnetic pull", so I went with that, although thinking about it traction does fit better in this sentence.